django/db/backends/utils.py

```diff
@@ -209,3 +209,16 @@ def format_number(value, max_digits, decimal_places):
     if decimal_places is not None:
         return "%.*f" % (decimal_places, value)
     return "{:f}".format(value)
+
+
+def strip_trailing_quotes(table):
+    """
+    Prepares table name for tables, triggers, sequences.
+    It processes case when db_table option for model Meta contains double-quoted table name.
+    Resulting name will be returned without double quotes despite of it had it or not before preparing.
+    """
+
+    if table.startswith('"') and table.endswith('"'):
+        return table[1:-1]
```

Comment:
If you declare model class like in ticket:
```python
class DjangoTestDocumento(models.Model):
    objectid = models.BigIntegerField(primary_key=True,unique=True)
    numero = models.CharField(max_length=150, blank=True, null=True)
    pessoas = models.ManyToManyField(DjangoTestPessoa)
    
    class Meta:
        db_table = '"GEO"."DJANGO_DOCUMENTO"'
```
then `opts.db_table` is equal to `"GEO"."DJANGO_DOCUMENTO"` e.q. in `django/db/models/fields/related.py` line [1551](https://github.com/django/django/blob/master/django/db/models/fields/related.py#L1551). Please add test with such case.